slack_sdk/models/blocks/block_elements.py

```diff
@@ -566,18 +567,20 @@ def __init__(
         https://api.slack.com/reference/block-kit/block-elements#image
 
         Args:
-            image_url (required): The URL of the image to be displayed.
             alt_text (required): A plain-text summary of the image. This should not contain any markup.
+            image_url: The URL of the image to be displayed.
+            slack_file: A Slack image file object that defines the source of the image.
```

Comment:
Should we state that either `image_url` or `slack_file` is required somewhere here?

Reply:
yeah, but in this sdk we don't mention such in comments